### DISCUSSION TITLE
Hacktoberfest listing

### DISCUSSION THREAD

**Discussion by marcopollivier** [simple-icons/simple-icons#7953] [General]
Hey, people. WDYT to add the tags to participate, as an OSS repository, of Hacktoberfest?

**Comment by RPicster** [comment]
I really try to hold back, but I think what happens here with the Hacktober in this repo is a very bad example of treating contributors and taking part in the Hacktober.

There were Issues with the `hacktoberfest` label open.
I opened four PRs for four of these Issues especially to take part in the Hacktober.
Three of them were merged already and they did not get a `hacktoberfest-accepted` label.
My comments in my PRs asking if the person that added the labels to the issues could take a look were marked as spam.

If you don't want to give contributors what they came for, why even put the `hacktoberfest` label on issues in the first place? Just remove it from the Issues then.

Here are my PRs:
https://github.com/simple-icons/simple-icons/pull/7941
https://github.com/simple-icons/simple-icons/pull/7940
https://github.com/simple-icons/simple-icons/pull/7938

For me this is really annoying and I will definitely not contribute in this repo again.

**Reply by LitoMore** [reply]
Hi @RPicster, please be patient. Our maintainers and reviewers maintain this project in our free time. I have already forwarded your request to our team but no response so far.

But please stop spamming those requesting messages in every pull request. Thanks.

<img width="821" alt="CleanShot 2022-10-15 at 02 51 47@2x" src="https://user-images.githubusercontent.com/8186898/195919977-be465fec-ea4b-44da-8110-3aee3f35a628.png">

**Reply by RPicster** [reply]
The point why I am so stern about this, is because my PRs were on issues that already have the Hacktoberfest label.

I don't ask for you to decide on something here, the decision was already made when some maintainer added those labels to the issues.

To get the hacktoberfest-accepted label in those PRs should happen with no questions asked.

On the link you provided it is about something else: someone opened an issue and requested/asked for the Hacktoberfest labels to be added.

**Reply by LitoMore** [reply]
cc @mondeja What do you think?

**Reply by mondeja** [reply]
> cc @mondeja What do you think?

I don't have a strong opinion about it as I'm not reviewing icons now. Up to you, reviewers team @simple-icons/icon-review 👀

**Reply by LitoMore** [reply]
I'd prefer to remove all existing `hacktoberfest` and `hacktoberfest-accepted` labels from our projects.

At first I patiently responded to those who said "please assign to me" in the issue message, but got no response. Until I found out they were all from the Hacktoberfest event.

**Reply by RPicster** [reply]
@mondeja @LitoMore 
You can remove them if you want no further engagement with people coming here for that. But you should assign `hacktoberfest-accepted` to those merged PRs for Issues that already had the `hacktoberfest` label when the PRs were merged and the issues were closed.

Otherwise I feel that it would be really unfair.

Of course it's ok to not take part in Hacktober, but if people contributed to issues that already *had* the label (and there were plenty in this repo), that would be unfair.

**Reply by LitoMore** [reply]
We are under no obligation to cater to Hacktoberfest every year. We have the right to opt out.

It should be passion, not t-shirts and stickers, that drive your contribution to the open-source community.

**Reply by RPicster** [reply]
> We are under no obligation to cater to Hacktoberfest every year. We have the right to opt out.
> 
> It should be passion, not t-shirts and stickers, that drive your contribution to the open-source community.

I still have the feeling you don't looked at the original issues...

The issues I contributed towards **already HAD the label** applied by one of this repos maintainers.
Having these labels on issues means that they are available for Hacktoberfest participants.

**I don't ask for you to take part - you already did that by having those labels:**
![image](https://user-images.githubusercontent.com/9423774/195982438-f848cd16-5dd5-4fd3-9559-120c4748123a.png)
These were the labels on all Issues where I opened up PRs.

My motivation is not just the shirt, it's also taking part and helping out on a repo where I can. But if those labels weren't already there in the first place I would not have taken the time to open the PRs here but somewhere else.

**Reply by LitoMore** [reply]
That label was attached last year. This year, I will remove all existing hacktoberfest-related labels from our projects.

**Reply by RPicster** [reply]
> That label was attached last year. This year, I will remove all existing hacktoberfest-related labels from our projects.

Oh no! I didn't notice that! I'm very sorry for being so stern about the whole topic...

I did in no way mean to be the asshole here, but I guess I was.

I hope you accept my apology, it was in no way done with bad intent. I just saw the label and wanted to take part.

**Reply by LitoMore** [reply]
@RPicster Never mind. Thank you for your contributions again!

**Comment by LitoMore** [comment]
~~I'd like to give +1 to Hacktoberfest. This can attract more contributors to help us submit missing icons or handle other users' icon requests.~~

But I must share my concern about Hacktoberfest, see this comment: https://github.com/sindresorhus/got/issues/1485#issuecomment-703132581.

And:

- https://twitter.com/search?q=hacktoberfest%20spam&src=typed_query&f=top
- https://twitter.com/shitoberfest

**Reply by marcopollivier** [reply]
I really understand the concern, but I don't think it should be used to harm anyone who wants to enjoy the event as extra motivation.

I've even gotten the habit of sharing more content from the OpenSource community thanks to Hacktoberfest.

I really don't agree with the argument in the previous comment "if I don't have the hashtag I don't contribute anymore". It's not allowed with the community spirit.

If the project decides not to support the event, I will respect the decision and I'll continue collaborating when necessary.

I have a draft-PR open (in my personal example) because I need to use the Backstage icon feature. Even if the project doesn't add the badge for me, I'll open it anyway. And this will not diminish my respect for the community it maintains, it will not impact my dedication to doing the project and I will seek to respect SimpleIcons' quality guidelines

**Reply by LitoMore** [reply]
@marcopollivier Thank you for pointing this out.
